elasticsearch.tf

```diff
@@ -64,3 +64,36 @@ resource "elasticstack_elasticsearch_index_template" "default" {
     })
   }
 }
+
+resource "elasticstack_elasticsearch_index_lifecycle" "default" {
+  count = module.elaticsearch_label.enabled ? 1 : 0
+  name  = "${local.elasticsearch_index_template_name}-policy" # name is predefined by fluentd's elasticsearch datastreams plugin
+
+  hot {
+    min_age = "0ms"
+    set_priority {
+      priority = 200
+    }
+    rollover {
+      max_primary_shard_size = var.elasticsearch_lifecycle_policy.hot_phase_max_primary_shard_size
+      max_age                = var.elasticsearch_lifecycle_policy.hot_phase_max_age
+    }
+  }
+
+  warm {
+    min_age = var.elasticsearch_lifecycle_policy.hot_phase_max_age
```

Comment:
this was now supposed to get its own variable, like var.elasticsearch_lifecycle_policy.warm_phase_min_age, wasn't it?